pyproject.toml

```diff
@@ -52,7 +52,7 @@ options = {"BUILD_TOKENIZERS" = "OFF"}
 [build-system]
 requires = [
     "py-build-cmake==0.3.4",
-    "openvino~=2025.1.0.0.dev",
```

Comment:
why do we need it?

Reply:
just added debug version by mistake, sorry, removed